.github/workflows/main-pr.yaml

```
@@ -0,0 +1,90 @@
+
+          echo "Padded Project version: $VERSION_PADDED"
+          echo "Padded Latest tag version: $LATEST_TAG_PADDED"
+
+          # Compare the padded versions
+          if [ "$VERSION_PADDED" -gt "$LATEST_TAG_PADDED" ]; then
+            echo "Project version $VERSION is greater than latest tag $LATEST_TAG_VERSION"
+            echo "create_tag=true" >> $GITHUB_ENV
+          else
+            echo "Project version $VERSION is not greater than latest tag $LATEST_TAG_VERSION"
+            echo "create_tag=false" >> $GITHUB_ENV
+            exit 1 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)